src/05-excitation-transfer.md

```diff
@@ -0,0 +1,762 @@
+
+
+How will we use `minimize_scalar`? We first need to provide a function that calculates the difference in the energies of two levels.
+
+```python
+def level_difference(DeltaE, omega, U, level_number):
+    H = DeltaE*two_state_1 + DeltaE*two_state_2 + omega*bosons + U*interaction_1 + U*interaction_2
+    evals, ekets = H.eigenstates()
+    return np.abs(evals[level_number] - evals[level_number-1])
+```
+
+We then pass this `level_difference` function to `minimise_scalar`. The first argument of `level_difference` (i.e. `DeltaE`) will be used by `minimize_scalar` as a variable and the rest will be considered as "arguments" (i.e. fixed constants). Finally, we must specify `bounds` (i.e. min and max `DeltaE`) inside of which we expect to find the anti-crossing.
+
+Let's give it a go using:
+- `omega = 1`
+- `U = 0.1`
+- `level_number = 5`
+- `bounds = [1.9, 2]`
+
+```python
+# oemga = 1
+# U = 0.1
+# level_number = 5 - this will allow us to calculate level_5/4 energy difference
+anti_crossing = minimize_scalar(level_difference, args=(1, 0.1, 5), method="Bounded", bounds=[1.9, 2])
+```
+
+```python
+anti_crossing
+```
+
+We see that the true anti-crossing occurs at $\Delta E = 1.9328593467366724$ - very precise  🤓 .
+
+We can now use this value to create the Hamiltonian for our simulation.
+
+> Feel free to adjust the value of  $\Delta E$ and see how sensitive the results are to the precise position of the anti-crossing.
+
+```python
+H = anti_crossing.x*two_state_1 + anti_crossing.x*two_state_2 + 1*bosons + 0.1*interaction_1 + 0.1*interaction_2
+```
+
+We want to start the system off in the $|0,+, + \rangle$ state - this should be the first state in the state list, let's check.
+
+```python
+nmm_list[0]
+```
+
+Great. We can now set up the initial state by using the `basis` function as we have done before.
+
+```python
+psi0 = basis(len(nmm_list), 0)
+```
+
+Now we are ready to simulate
+
+```python
+times = np.linspace(0.0, 100000.0, 10000)
+P, psi = simulate(H, psi0, times)
+```
+
+We need to recreate the bra-ket labels because we are now only working with even parity states.
+
+```python
+bra_labels, ket_labels = make_braket_labels(nmm_list)
+```
+
+Now let's plot the results using a helper function `plot_prob` that pulls together plotting code that we used in the last tutorial.
+
+```python
+plot_prob(P, times, ket_labels)
+plt.title(f"2 TSS with {H_latex}  ($\Delta E \\approx 1.9328$, $\omega=1$, $U=0.1$)   (Fig 8)");
+```
+
+Fig 8 shows exactly what we expected i.e. both TSS transition from "+" to "-" each giving of 2 bosons in the process, i.e. $|0,+, + \rangle \rightarrow |4,-, - \rangle$.
```

Comment:
I would just mention that this is Down conversion if you agree. Maybe also in the title of the section